test/protobuf/decoder_test.exs

```diff
@@ -126,7 +126,7 @@ defmodule Protobuf.Decoder.Test do
       use Protobuf, from: Path.expand("../proto/mumble.proto", __DIR__)
     end
 
-    msg = MumbleProto.Authenticate.new(username: "bitwalker")
+    msg = MumbleProto.MumbleProto.Authenticate.new(username: "bitwalker")
```

Comment:
This doesn't seem to be working anymore, however the test for it was using `Module.create` which does still work, how that's possible is beyond me, but in any case, it appears this broke some time in the past. I can assure you though that how I described it above is indeed how it worked (and if you look at the code for injected modules, you'll see why that's the case). Looks like I have some `git bisect` in my future.

`defs` doesn't actually contain any information about the generated module, only about the protobuf messages/fields/enums/etc. that are part of that module. I'm aware of the `:module_name` convention circumventing the `Elixir.` prefix, but I meant modules defined with the convention we all use in Elixir by default (i.e. Pascal-cased, implicit atoms, instead of colon-prefixed explicit atoms).